python/hail/expr/aggregators.py

```diff
@@ -569,8 +569,12 @@ def stats(expr):
     agg = _to_agg(expr)
     if not is_numeric(agg._type):
         raise TypeError("'stats' expects a numeric argument, found '{}'".format(agg._type))
-    return _agg_func('stats', agg, tstruct(['mean', 'stdev', 'min', 'max', 'nNotMissing', 'sum'],
-                                           [tfloat64, tfloat64, tfloat64, tfloat64, tint64, tfloat64]))
+    return _agg_func('stats', agg, tstruct(mean=tfloat64,
+                                           stdev=tfloat64,
+                                           min=tfloat64,
+                                           max=tfloat64,
+                                           nNotMissing=tint64,
```

Comment:
`n_present > n_not_missing > nNotMissing` but perhaps outside scope

Reply:
definitely outside scope. This involves scala changes.